src/app/examples/list/no-shape/list-no-shape-example.component.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../../kirby/scss/utils";
+
+kirby-card {
+  margin: 5px;
+  width: 330px;
+
+  @include media(">=large") {
+      width: 500px;
+  }
+
+  kirby-card-header {
+      padding: size('xxxs');
```

Comment:
Nitpicking: You have 4 spaces per tap here but 2 spaces other places in the file.
And all the other code uses 2 spaces. 😃

Reply:
Doh.. my VS Code stopped auto formatting for some reason. Will fix :-)

Reply:
Fixed manually. My Prettier is still broken for some reason.